src/foam/u2/view/ColumnConfigView.js

```diff
@@ -575,6 +575,11 @@ foam.CLASS({
   package: 'foam.u2.view',
   name: 'SubColumnSelectConfig',
   extends: 'foam.u2.View',
+
+  imports: [
+    'auth'
```

Comment:
Should be 'auth?', since not all FOAM users / usages have an auth service installed and this will break them.